src/routes/userRoute.js

```diff
@@ -1,4 +1,5 @@
 const userCtrl = require('../controllers/userCtrl');
+// const authMiddleware = require('../middleware/authMiddleware');
```

Comment:
Why it's commented for now ?

Reply:
Was for testing, and i think will implement here to so i didn't remove it. we can if you want

Reply:
Okay you can let it commented in that case :)